### PR TITLE
Graph Page updates

### PR DESCRIPTION
**Pull request by mikestone14**

### CODE REVIEW

File: views/components/Matrix.d3.js [hunk 1]
```diff
@@ -300,155 +521,239 @@ export default class Matrix extends AbstractGraph {
...
+      .style('fill', (d, i) => {
+        return d.marker == 'N/A' ? '#ffffff' : '#545454'
+      })
+      .on('click', (d, i) => {
         let day = d.day
-        let link = '/deepdive/' + this.props.study + '/' + this.props.subject + '/' + day
+        let link =
+          '/deepdive/' + this.props.study + '/' + this.props.subject + '/' + day
         window.open(`${basePath}${link}`, '_blank')
       })
```
Comment (Elkrival) [inline views/components/Matrix.d3.js:632]: I think we can remove this. The page is being unused https://gnardog.slack.com/archives/C03KGHSSTNC/p1688340722944729?thread_ts=1687804987.105369&cid=C03KGHSSTNC
```suggestion

```

File: views/components/Matrix.d3.js [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-import { stringToDate } from '../../server/utils/dateConverter';
-import basePathConfig from '../../server/configs/basePathConfig';
+import * as d3 from 'd3'
+import AbstractGraph from './AbstractGraph.d3'
+import { getColor } from '../../server/utils/invertColor'
+import { stringToDate } from '../../server/utils/dateConverter'
+import basePathConfig from '../../server/configs/basePathConfig'
 
-const basePath = basePathConfig || '';
+const basePath = basePathConfig || ''
```
Comment (Elkrival) [inline views/components/Matrix.d3.js:7]: The deepdive page is unused we might not need this. 
https://gnardog.slack.com/archives/C03KGHSSTNC/p1688340722944729?thread_ts=1687804987.105369&cid=C03KGHSSTNC

File: views/routes/routes.js [hunk 3]
```diff
@@ -8,9 +8,11 @@ export const routes = {
   basePath,
   home: `${basePath}/`,
   userAccount: '/user-account',
-  configs: `/configs`,
+  configs: '/configs',
   editConfigPage: '/config/:config_id/edit',
-  dashboard: `${basePath}/dashboard`,
+  dashboards: `${basePath}/dashboard`,
```
Comment (Elkrival) [inline views/routes/routes.js:13]:
```suggestion
```

File: views/pages/GraphPage.jsx [hunk 4]
```diff
@@ -57,133 +48,59 @@ const socket = io(socketAddress, {
...
-    return listItem
-  }
-  downloadPng = () => {
-    let SID = this.state.subject.sid
-    this.canvas.toBlob((blob) => {
-      FileSaver.saveAs(blob, SID + '.png')
+const GraphPage = () => {
+  const { user, classes, theme } = useOutletContext()
+  const el = React.useRef()
+  // el.current --> DOM Node || undefined
```
Comment (Elkrival) [inline views/pages/GraphPage.jsx]:
```suggestion
```

File: views/pages/GraphPage.jsx [hunk 5]
```diff
@@ -27,27 +26,19 @@ import DialogContent from '@material-ui/core/DialogContent'
...
 import basePathConfig from '../../server/configs/basePathConfig'
-import { routes, apiRoutes } from '../routes/routes'
-import { withRouter } from '../hoc/withRouter'
+import { apiRoutes } from '../routes/routes'
+import Matrix from '../components/Matrix.d3'
+import GraphPageTable from '../components/GraphPageTable'
+import api from '../api'
 
 const basePath = basePathConfig || ''
+const cardSize = 50
```
Comment (Elkrival) [inline views/pages/GraphPage.jsx]:
```suggestion
const cardSize = 20
```

**Review by Elkrival — APPROVED** [review]
LGTM

File: views/components/Matrix.d3.js (hunk 2 above)
Comment (mikestone14) [reply views/components/Matrix.d3.js:7]: for posterity - we discussed this and decided to leave it for now until we decide to deliberately remove it.

**Pull request merged**